lib/twitch.rb

```diff
@@ -33,7 +33,7 @@ def self.followed_ids(login)
 
     class << self
       def get(login)
-        route(login).get(Twitch.headers)
+        route(login.twitch_id).get(Twitch.headers)
```

Comment:
Wow haha ok that was totally messed up then, even though it technically still worked. Yeah if anything is a user object but is called `login` something is definitely amiss. Let's change to ID.